sensu/config_test.go

```diff
@@ -0,0 +1,113 @@
+package sensu
+
+import (
+	"os"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/upfluence/sensu-client-go/Godeps/_workspace/src/github.com/upfluence/sensu-go/sensu/check"
+	stdClient "github.com/upfluence/sensu-client-go/Godeps/_workspace/src/github.com/upfluence/sensu-go/sensu/client"
+)
+
+func validateStringParameter(
+	actualRabbitMqUri string,
+	expectedRabbitMqUri string,
+	parameterName string,
+	t *testing.T) {
+
+	if actualRabbitMqUri != expectedRabbitMqUri {
+		t.Errorf("Expected "+parameterName+
```

Comment:
if you use `Errorf` also use a string formatter (` "%s" `) to include `parameterName` in the output